pkgs/applications/file-managers/clifm/default.nix

```diff
@@ -21,11 +25,11 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
-  meta = with lib; {
+  meta = {
```

Comment:
nixpkgs convention is to leave it as it currently is, otherwise looks good to me

Reply:
Fixed,
I've also updated to the last version 1.12 instead of 1.11

Thanks so much @legendofmiracles for your nice review!